lib/src/chart/bar_chart/bar_chart_renderer.dart

```diff
@@ -9,17 +9,27 @@ import 'package:flutter/cupertino.dart';
 
 /// Low level BarChart Widget.
 class BarChartLeaf extends LeafRenderObjectWidget {
-  const BarChartLeaf({super.key, required this.data, required this.targetData});
+  const BarChartLeaf({
+    super.key,
+    required this.data,
+    required this.targetData,
+    required this.canBeScaled,
+    required this.boundingBox,
```

Comment:
Here we have `boundingBox` and in the axis_chart_scaffold_widget we have `chartRect`
But as I understood they're the same and they represent the virtual size that the chart is being drawn on.
So I think it makes sense to use a unique name for this concept. Because it will be confusing with the actual chart size that the user sees.

Why not using `chartVirtualSize`?

Reply:
You are right, the naming is not consistent.

`chartVirtualSize` is a step in the right direction. Maybe `chartVirtualRect` would be an even better name as it's a `Rect`, i.e. Offset and Size.

What do you think?

Reply:
Yes, `chartVirtualRect` makes sense. So the name is already descriptive, but we can add a bit of explanation in the code comment.

Reply:
done ✅